webapp/src/main/java/com/box/l10n/mojito/service/tm/GenerateMultiLocalizedAssetJob.java

```diff
@@ -0,0 +1,60 @@
+package com.box.l10n.mojito.service.tm;
+
+import com.box.l10n.mojito.quartz.QuartzJobInfo;
+import com.box.l10n.mojito.quartz.QuartzPollableJob;
+import com.box.l10n.mojito.quartz.QuartzPollableTaskScheduler;
+import com.box.l10n.mojito.rest.asset.LocaleInfo;
+import com.box.l10n.mojito.rest.asset.LocalizedAssetBody;
+import com.box.l10n.mojito.rest.asset.MultiLocalizedAssetBody;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class GenerateMultiLocalizedAssetJob
+    extends QuartzPollableJob<MultiLocalizedAssetBody, MultiLocalizedAssetBody> {
+
+  @Autowired QuartzPollableTaskScheduler quartzPollableTaskScheduler;
+
+  @Override
+  public MultiLocalizedAssetBody call(MultiLocalizedAssetBody input) throws Exception {
+
+    for (LocaleInfo localeInfo : input.getLocaleInfos()) {
+      String outputTag =
+          localeInfo.getOutputBcp47tag() != null
+              ? localeInfo.getOutputBcp47tag()
+              : localeInfo.getBcp47Tag();
+      QuartzJobInfo<LocalizedAssetBody, LocalizedAssetBody> quartzJobInfo =
+          QuartzJobInfo.newBuilder(GenerateLocalizedAssetJob.class)
+              .withInlineInput(false)
+              .withParentId(getParentId())
```

Comment:
Can we, thanks to this, show a status in the CLI console of all tasks while polling on the main task?